gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/mapper/UserMapper.java

```diff
@@ -93,6 +95,22 @@ public static User convert(io.gravitee.am.model.User user, String baseUrl, boole
         meta.setResourceType(User.RESOURCE_TYPE);
         meta.setLocation(baseUrl + (listing ? "/" + scimUser.getId() : ""));
         scimUser.setMeta(meta);
+
+        // Gravitee User extension schemas
+        // To determine if the current user resource is a Gravitee User Resource
+        // we remove every OpenID standard claims from the additionalInformation map
+        // if we have remaining claims, the user can be set up as a Gravitee User Resource
+        Map<String, Object> customClaims =
```

Comment:
Sorry for missing that point in previous review but we should filter the `op_id_token` & `op_access_token` like we did for IDToken otherwise a user linked to an external IDP may expose the access_token provided by the OP.

https://github.com/gravitee-io/gravitee-access-management/blob/master/gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oidc/service/idtoken/impl/IDTokenServiceImpl.java#L214